runtime/polkadot/constants/src/lib.rs

```diff
@@ -22,6 +22,9 @@ pub use self::currency::DOLLARS;
 pub mod currency {
 	use primitives::v0::Balance;
 
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9010).
+	pub const EXISTENTIAL_DEPOSIT: Balance = 10 * CENTS;
```

Comment:
These values are also not correct and the docs are also not correct.

Reply:
used the values from cumulus, updated now